app/src/lib/release-notes.ts

```diff
@@ -83,10 +83,23 @@ export function getReleaseSummary(
   }
 }
 
-export async function getChangeLog(): Promise<ReadonlyArray<ReleaseMetadata>> {
+export async function getChangeLog(
+  limit?: number
+): Promise<ReadonlyArray<ReleaseMetadata>> {
   const changelog =
     'https://central.github.com/deployments/desktop/desktop/changelog.json'
-  const query = __RELEASE_CHANNEL__ === 'beta' ? '?env=beta' : ''
+  const params = []
+  if (__RELEASE_CHANNEL__ === 'beta') {
+    params.push('env=beta')
+  }
+  if (limit !== undefined) {
+    params.push(`limit=${limit}`)
+  }
+  let query = ''
+  params.forEach(param => {
+    const prefix = query === '' ? '?' : '&'
+    query += `${prefix}${param}`
+  })
```

Comment:
I sorta think this might be better off using [URLSearchParams, see MDN](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)

```suggestion
  const params = new URLSearchParams()
  if (__RELEASE_CHANNEL__ === 'beta') {
    params.append('env', 'beta')
  }
  if (limit !== undefined) {
    params.append('limit', limit)
  }
  let query = `?{params.toString()}'
```

Reply:
Dude! Did not know we had this URLSearchParams (bad assumption based on what it was before)! Thank you for pointing it out.

Reply:
Refactored! Thanks again